### PR TITLE
[14.0][MIG] hr_employee_calendar_planning

### PR DESCRIPTION
**Pull request by aphon61bank**
Standard Migration, Ready for review.

### CODE REVIEW

**Review by fshah-initos — APPROVED** [review]
Looks good to me!

**Comment by fshah-initos** [comment]
If somone could add reviews here we can merge this PR soon.

**Comment by fshah-initos** [comment]
It would be great if somone could add reviews so that we can merge the PR soon.

**Review by kittiu — APPROVED** [review]
No major change. LGTM.

**Comment by fshah-initos** [comment]
It would be great if maintainers can merge the PR.

File: hr_employee_calendar_planning/models/hr_employee.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+                        },
+                    )[0]
+                    seq += 1
+                    vals_list += [(0, 0, data)]
+        if two_weeks:
+            SECTION_LINES[0][2]["sequence"] = -seq
+            vals_list = SECTION_LINES + vals_list
+        self.resource_calendar_id.attendance_ids = vals_list
+
+    def regenerate_calendar(self):
```
Comment (etobella) [inline hr_employee_calendar_planning/models/hr_employee.py:108]: This function is not tested, but I think it is not used. Might it be?

**Pull request merged**